aiohttp/connector.py

```diff
@@ -813,10 +813,13 @@ def _get_fingerprint(self, req):
         fingerprint = self._get_fingerprint(req)
 
         try:
-            hosts = await self._resolve_host(
+            # Cancelling this lookup should not cancel the underlying lookup
+            #  or else the cancel event will get broadcast to all the waiters
+            #  across all connections.
+            hosts = await  asyncio.shield(self._resolve_host(
                 req.url.raw_host,
                 req.port,
-                traces=traces)
+                traces=traces))
```

Comment:
Please pass `loop=self._loop` into `shield()`.
Consider it as tiny speed optimization.

Reply:
fixed